database/faker/FakerRockTheVoteReportRow.php

```diff
@@ -25,11 +25,21 @@ public function rockTheVoteReportRow($data = [])
             'Phone' => $this->generator->phoneNumber,
             'Finish with State' => 'Yes',
             'Pre-Registered' => 'No',
-            'Started registration' => Carbon::now()->format('Y-m-d H:i:s O'),
+            'Started registration' => $this->rockTheVoteStartedRegistration(),
             'Status' => 'Step 1',
             'Tracking Source' => 'ads',
             'Opt-in to Partner email?' => 'Yes',
             'Opt-in to Partner SMS/robocall' => 'Yes',
         ], $data);
     }
+
+    /**
+     * Return a mock Started registration field value.
+     *
+     * @return string
+     */
+    public function rockTheVoteStartedRegistration($numDaysToSubtract = 0)
+    {
+        return Carbon::now()->subDays($numDaysToSubtract)->format('Y-m-d H:i:s O');
```

Comment:
Is the main reason for this generator the non-standard date-format? Wondering if `daysAgoInRockTheVoteFormat()` might read a little clearer? If only they used normal ISO-8601!

Reply:
Yep, that's the only reason for the non-standard and I don't mind renaming. It wouldn't really break any of our tests to mock with normal ISO-8601, but seemed better to try to mock the data to match as best as possible.